pkgs/by-name/wm/wmidump/package.nix

```
@@ -0,0 +1,43 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+}:
+
+stdenv.mkDerivation {
+  pname = "wmidump";
+  # Since the project does not specify a version,
+  # so we'll use the commit number as one.
+  version = "17";
+
+  src = fetchFromGitHub {
+    owner = "iksaif";
+    repo = "wmidump";
+    rev = "ae728927dd5b32d965a25660bb68a4c781eb5103";
+    sha256 = "sha256-BFNZSpa70lzw52qT6+sB3tRUgCq9e1XmVyxh7J+AjQk=";
+  };
+
+  patches = [
+    ./0001-fix-cross-compilation.patch
+  ];
+
+  installPhase = ''
+    runHook preInstall
+
+    install -D -m 755 wmidump $out/bin/wmidump
+
+    runHook postInstall
+  '';
+
+  meta = with lib; {
+    # Description is taken _sic erat scriptum_ from GitHub repo description,
+    # grammar preserved.
+    description = "Dump WMI informations from ACPI tables";
+    mainProgram = "wmidump";
+    homepage = "https://github.com/iksaif/wmidump";
+    maintainers = with maintainers; [ synalice ];
+    # There is no license. I've created an issue about it upstream:
+    # https://github.com/iksaif/wmidump/issues/5
+    # license = licenses.???;
```

the correct protocol in this scenario is:
```suggestion
    license = lib.licenses.unfree;
```